src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Operators.cs

```diff
@@ -2587,34 +2588,45 @@ private bool GetBinopKindAndFlags(ExpressionKind ek, out BinOpKind pBinopKind, o
             return true;
         }
 
-        private static bool isDivByZero(ExpressionKind kind, EXPR op2)
+        private static bool IsDivByZero(ExpressionKind kind, EXPR op2, PredefinedType ptOp)
```

Comment:
Ah. I had been considering the case of floating-point values, which of course shouldn't throw, but when seeing that it is never hit with those I removed the guard against it but not the parameter.

Updated.